memory-store/migrations/000035_enhanced_indices.down.sql

```diff
@@ -0,0 +1,63 @@
+---- Transitions
+
+-- Drop new indices
+DO $$
+BEGIN
+  DROP index if exists idx_transitions_execution_id;
+
+  DROP index if exists idx_executions_created_at_covering;
+
+  DROP index if exists idx_transitions_created_at;
+END $$;
+
+-- Temporarily disable compression (for unique index creation)
+ALTER TABLE transitions SET (timescaledb.compress = false);
```

Comment:
why is this needed?

Reply:
since we drop the unique index later, might as well skip it here. compress<>decompress is an expensive operation. perhaps you can keep these but comment them out explaining why